week3/yourChoice/src/components/Content/Content.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { ChooseContainer } from './style/ContentStyles';
+import ContentWrapper from '../Layout/ContentWrapper';
+import ContentHeader from '../Layout/ContentHeader';
+import H1 from '../UI/H1';
+import ChooseBtn from '../UI/ChooseBtn';
+import Button from '../UI/Button';
+import Prefer from './Prefer';
+
+// 전체 content div
+const Content = (props) => {
+  return (
+    <ContentWrapper>
+      <ContentHeader>
+        <H1>원하는 추천 방식을 골라줘!</H1>
+      </ContentHeader>
+
+      {props.isStart === '' && (
+        <ChooseContainer>
+          <ChooseBtn
+            onClick={() => {
+              props.isStartHandler('취향대로 추천');
+            }}
+          >
```

Comment:
이전 과제들에서 button 태그에 type="button"을 지정해줘야 한다는 내용 기억하나용!? 
Styled component를 만들어서 사용할 때도 이러한 속성들은 잊지않고 챙겨줘야합니다!! 
대표적으로 
- button -> type="button" 
- img -> alt="대체 텍스트"
이런 친구들이 있습니다!! 
```suggestion
          <ChooseBtn
            type="button"
            onClick={() => {
              props.isStartHandler('취향대로 추천');
            }}
          >
```